docs/en/source/quickstart/install.md

```diff
@@ -1,3 +1,86 @@
-# Installation
+# Install TinyMS
 
-TODO
+## User install TinyMS
+
+### Pypi
+
+For users owning a clean environment, recommend using [pypi](https://pypi.org/) to install TinyMS after satisfy the following prerequisites. For setting up python environment, [Anaconda](https://www.anaconda.com/products/individual#Downloads) is a good choice.
+
+Prerequisites  
+
+- Ubuntu: `18.04`
+- Python: `3.7.5`
+
+Chinese Domestic users recommend running the following code setting the mirror site of pypi to solve the downloading issue
+
+```shell
+mkdir -pv /root/.pip \
+&& echo "[global]" > /root/.pip/pip.conf \
+&& echo "trusted-host=mirrors.aliyun.com" >> /root/.pip/pip.conf \
+&& echo "index-url=http://mirrors.aliyun.com/pypi/simple/" >> /root/.pip/pip.conf
+```
+
+```shell
+pip install tinyms==0.1.0
+```
+
+### Docker
+
+For users' environment do not want to change the local env which also cannot satisfy the prerequisites, using [docker](https://www.docker.com/) to install is recommended
```

Comment:
For those who don't want to affect the local develop environment due to difficulty of meeting the prerequisites